### ISSUE TITLE
The balloon tiles more than once with a horse outfit

### ISSUE THREAD

**Issue by cosminhume** [arianne/stendhal#257]
## 🐞 Describe the bug

During Mine Town Revival Weeks, one can rent a horse suit from Liliana and get a balloon from Maltos. The two decorations don't look together. See the screenshots below:
![horse_balloon_front](https://user-images.githubusercontent.com/42698386/67139161-91934300-f255-11e9-9687-9a79b7efd366.png)
![horse_balloon_right](https://user-images.githubusercontent.com/42698386/67139162-91934300-f255-11e9-8a49-ae0301694d04.png)
![horse_balloon_back](https://user-images.githubusercontent.com/42698386/67139160-91934300-f255-11e9-96d6-ad6f12abb8e6.png)

## 🔥 How to reproduce

1. Make sure Mine Town Revival Weeks are on
2. Go to Liliana in Semos
3. Hire a horse outfit
4. Go to Maltos in Semos Mountain N2 (near the mine entrance)
5. Solve the puzzle and get a balloon
6. Look at the horse with the balloon, from all sides

> [09:34] &lt;cosmin&gt; hi
[09:34] &lt;Liliana&gt; Hi! How may I help you?
[09:34] &lt;cosmin&gt; offer
[09:34] &lt;Liliana&gt; Just tell me if you want to hire a gown, hire a green dress, hire glasses, hire other glasses, hire a hat, hire an alien suit, hire a horse outfit, hire a girl horse outfit, hire a jumpsuit, hire dungarees, hire a bunny suit or hire an orange outfit.
[09:35] &lt;cosmin&gt; hire horse
[09:35] &lt;Liliana&gt; To hire a horse will cost 1200. Do you want to hire it?
[09:35] &lt;cosmin&gt; yes
[09:35] &lt;Liliana&gt; Thanks! You can wear this for 10 hours.
[09:35] &lt;cosmin&gt; bye
[09:35] &lt;Liliana&gt; Bye!

> [09:38] &lt;cosmin&gt; hi
[09:38] &lt;Maltos&gt; Hi, welcome to our small game of nine switches. Your task is to make all arrows point to the right. Easy? Well, there is a catch.
[09:38] &lt;cosmin&gt; catch
[09:38] &lt;Maltos&gt;  Each switch is linked to its neighbour and will change them as well. You have one minute to solve the puzzle. Do you want to play?.
[09:38] &lt;cosmin&gt; yes
[09:38] &lt;Maltos&gt; Good luck.
[09:38] &lt;Maltos&gt; Congratulations, cosmin you won! Here take this balloon.
[09:38] &lt;cosmin&gt; bye
[09:38] &lt;Maltos&gt; It was nice to meet you.

## 🌍  Environment

[x] Java client
[ ] Web client
Location in game: Semos, then Semos Mountain N2

## 🔭 Additional information

I would assume this to be due to the wider size of the horse, which would cause the balloon tile to appear more than once.

**Comment by AntumDeluge** [comment]
Yes, that is exactly the problem. The new horse images are larger than the old ones but the balloons are made for a 48x64 sprite. We truly new "mounts" in the game that use their own separate sprite.
